src/Provider.tsx

```diff
@@ -508,7 +508,12 @@ export const ThirdwebSDKProvider: React.FC<
 /**
  *
  * @returns {@link ThirdwebSDK}
- * @internal
+ * Create an instance of the Thirdweb SDK that uses the connected wallet to sign transactions,
```

Comment:
this does not *create* an instance of the SDK but rather it accesses the SDK instance created by `<ThirdwebProvider>`

Reply:
```suggestion
 * Access the instance of the thirdweb SDK created by the ThirdwebProvider, that uses the connected wallet to sign transactions,
```